plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeHistoryTable.java

```diff
@@ -97,7 +114,18 @@ public ConnectorTableMetadata getTableMetadata()
     public RecordCursor cursor(ConnectorTransactionHandle transactionHandle, ConnectorSession session, TupleDomain<Integer> constraint)
     {
         TimeZoneKey timeZoneKey = session.getTimeZoneKey();
-        Iterable<List<Object>> records = Iterables.transform(commitInfoEntries, commitInfoEntry -> getRecord(commitInfoEntry, timeZoneKey));
+        TrinoFileSystem fileSystem = fileSystemFactory.create(session);
+        Iterable<List<Object>> records = () ->
+                stream(new DeltaLakeTransactionLogIterator(
+                        fileSystem,
+                        new Path(metastore.getTableLocation(tableName, session)),
+                        Optional.empty(),
+                        Optional.empty()))
```

Comment:
Shouldn't these 4 lines be more indented ?

Reply:
I haven't found a better way to highlight the stream. Could you pls suggest a change in this regard?